src/services/server-response-handler.js

```diff
@@ -0,0 +1,26 @@
+const logger = require('./logger');
+const errorMessages = require('../utils/error-messages');
+const P = require('bluebird');
+
+function ServerResponseHandler(error, result) {
+  this.perform = () => {
+    return P.try(() => {
+      if (result.status === 200 && result.body && result.body.data && result.body.data.attributes) {
+        return result.body.data;
+      } else {
+        if (result.status === 0) {
+          logger.error(errorMessages.SESSION.SERVER_DOWN);
```

Comment:
If it is the `server-response-handler` the error messages are related to sessions.